VideoLocker/res/layout/row_discussion_thread.xml

```diff
@@ -23,67 +27,63 @@
             android:layout_marginTop="@dimen/x_small_icon_margin"
-            android:gravity="center_vertical"
+            android:gravity="top"
             android:orientation="horizontal">
 
             <org.edx.mobile.view.custom.IconImageViewXml
                 android:id="@+id/discussion_post_closed_icon"
-                android:layout_width="@dimen/edx_x_small"
-                android:layout_height="@dimen/edx_x_small"
-                android:layout_marginEnd="@dimen/discussion_posts_icon_margin"
-                android:layout_marginRight="@dimen/discussion_posts_icon_margin"
-                app:iconColor="@color/edx_grayscale_neutral_base"
+                style="@style/discussion_thread_icon"
+                app:iconColor="@color/edx_grayscale_neutral_dark"
                 app:iconName="fa-lock" />
 
             <org.edx.mobile.view.custom.IconImageViewXml
                 android:id="@+id/discussion_post_pin_icon"
-                android:layout_width="@dimen/edx_x_small"
-                android:layout_height="@dimen/edx_x_small"
-                android:layout_marginEnd="@dimen/x_small_icon_margin"
-                android:layout_marginRight="@dimen/x_small_icon_margin"
-                app:iconColor="@color/edx_grayscale_neutral_base"
+                style="@style/discussion_thread_icon"
+                app:iconColor="@color/edx_grayscale_neutral_dark"
                 app:iconName="fa-thumb-tack" />
 
             <org.edx.mobile.view.custom.IconImageViewXml
                 android:id="@+id/discussion_post_following_icon"
-                android:layout_width="@dimen/edx_x_small"
-                android:layout_height="@dimen/edx_x_small"
-                android:layout_marginEnd="@dimen/x_small_icon_margin"
-                android:layout_marginRight="@dimen/x_small_icon_margin"
-                app:iconColor="@color/edx_grayscale_neutral_base"
+                style="@style/discussion_thread_icon"
+                app:iconColor="@color/edx_grayscale_neutral_dark"
                 app:iconName="fa-star" />
 
             <TextView
-                android:id="@+id/discussion_post_author"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:textColor="@color/edx_grayscale_neutral_base"
-                android:textSize="@dimen/edx_x_small"
-                tools:text="By Staff" />
+                android:id="@+id/discussion_post_replies_count"
+                style="@style/discussion_responses_small_text"
+                android:layout_marginEnd="@dimen/discussion_posts_icon_margin"
+                android:layout_marginRight="@dimen/discussion_posts_icon_margin"
+                tools:text="| Total: 20" />
+
+            <TextView
+                android:id="@+id/discussion_post_date"
+                style="@style/discussion_responses_small_text"
+                tools:text="| Last post: Apr 06, 2016" />
```

Comment:
Hmm, the pipe character already gets read as 'Vertical Bar'. I suppose you could just merge the two text views (with a custom content description that removed the pipe), and leave the icons as they are. Note that there is no content description on the icons at the moment. We should probably open an accessibility ticket on that if not created already.